src/main/java/crawlercommons/sitemaps/SiteMap.java

```diff
@@ -93,10 +91,17 @@ private void setUrl(String url) {
     }
 
     public String toString() {
-        StringBuilder sb = new StringBuilder();
-
-        sb.append("url = \"").append(url).append("\", lastMod = ").append((getLastModified() == null) ? "null" : SiteMap.W3C_FULLDATE_FORMATTER_UTC.format(getLastModified().toInstant()))
-                        .append(", type = ").append(getType()).append(", processed = ").append(isProcessed()).append(", urlListSize = ").append(urlList.size());
+        StringBuilder sb = new StringBuilder()
```

Comment:
Formatting is done by `mvn java-formatter:format` which would again put all the lines together. I agree (my personal preference) that every append in its own line is more readable. If ` //` is added to every line the formatter will keep it.

Reply:
Thanks, nice hack, I will try it then run the mvn formatter command and check to see the results.